CONTRIBUTING.rst

```diff
@@ -167,3 +167,30 @@ To speed up the integration process, it helps if on GitHub you `allow
 maintainers to edit your pull request
 <https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/>`_
 so they can fix small issues autonomously.
+
+.. rubric: Review guidelines
+
+A pull request will probably be reviewed by someone in a different location,
+maybe even a different time zone. Therefore, avoiding blockings and
+misunderstandings is key. The following ideas could help ease and speed
+up the review process:
+
+- Be clear when something is ready to reviewed (see `WIP` above).
+- If a PR requires changes, say so explicitly, but politely (use emojis!).
+- When something is ready to be merge say it explicitly. If the reviewer does
+not have permission to merge, it should ping the corresponding person.
```

Comment:
```suggestion
not have permission to merge, they should ping the corresponding person.
```
:wink: 
This is the new gender neutral pronoun... which is actually quite interesting in French because no pronouns like it existed and it had to be invented in the last few years! ( "Ile" I think is the recommended one in French)

Reply:
Oh nice, so for future reference, is "they" not "it" even if the intention is for a user (singular, not plural)

Reply:
Yes. In any case, "it" is for a singular *thing* in English, so we would have used "he or she" in the past (why the winking face :smile:  ), but now "they" is more appropriate... which is also interesting because using "they" like here used to be a grammatical mistake. "They" was solely considered plural, so using it to refer to a singular person was erroneous (but it was common slang, so it was a typical way to lose points in English classes!).

Reply:
Minor: I would honestly go for `she/he` or `(s)he`, they sounds weird :)

Reply:
minor: So... This can be a fraught topic and I am not necessarily the best champion for that cause... but "they" is now a [grammatically correct](https://owl.purdue.edu/owl/general_writing/grammar/pronouns/gendered_pronouns_and_singular_they.html) way to denote both "he or she" *and* non-binary subjects. Agreed, it does sound strange to use "they" when you are used to being prevented from saying it :)! If members of our community identify as such, it is not a big burden on us to use that pronoun to have that extra layer of inclusion. For me, it's just easier to use "they" than alternatives ;).

In any case, we have been approaching this in good faith and no matter the decision the choice was not meant to exclude. You can merge when you want.